libhoney/client.py

```diff
@@ -145,6 +171,7 @@ def send_now(self, data):
         '''
         ev = self.new_event()
         ev.add(data)
+        self.log("enqueuing event ev = %s", ev.fields())
```

Comment:
I'd like to see a subtle difference in phrasing so that a search for the exact log message can differentiate the use of `send` from `send_now`